src/Compute/Compute/ChangeLog.md

```diff
@@ -26,6 +26,7 @@
 * Removed unversioned and outdated images from New-AzVmss `-ImageName` argument completers.
 * Added new parameters `-ElasticSanResourceId` and `-OptimizedForFrequentAttach` to `New-AzDiskConfig` cmdlet.
 * Added new parameter `-ElasticSanResourceId` to `New-AzSnapshotConfig` cmdlet.
+* Fixed `New-AzVmss` and `New-Azvm` to use `SharedGalleryImageId` parameter.
```

Comment:
Can you add a description of what this change is and what it is doing and what the problem was? TO the PR description I mean.